wfe/web-front-end.go

```diff
@@ -78,6 +81,28 @@ func (wfe *WebFrontEndImpl) HandlePaths() {
 
 // Method implementations
 
+func (wfe *WebFrontEndImpl) Index(response http.ResponseWriter, request *http.Request) {
+	// http://golang.org/pkg/net/http/#example_ServeMux_Handle
```

Comment:
Patterns cannot be duplicated (http://golang.org/pkg/net/http/#ServeMux.Handle): "If a handler already exists for pattern, Handle panics."

Secondly, http://golang.org/pkg/net/http/#ServeMux: "Longer patterns take precedence over shorter ones, so that if there are handlers registered for both "/images/" and "/images/thumbnails/", the latter handler will be called for paths beginning "/images/thumbnails/" and the former will receive requests for any other paths in the "/images/" subtree."

Hence, I believe that the order of `HandleFunc` calls does not matter at all.